examples/gym/gym_state.py

```diff
@@ -1,61 +1,57 @@
+import base64
 from copy import deepcopy
-from typing import Sequence, Tuple
+import logging
+import pickle
+from typing import Sequence
 
+
+import numpy as np
 import gym
 
 from rlmolecule.tree_search.graph_search_state import GraphSearchState
 
 from alphazero_gym import AlphaZeroGymEnv
 
+logger = logging.getLogger(__name__)
+
 
 class GymEnvState(GraphSearchState):
```

Comment:
would we want to add these classes to the main rlmolecule package?

Reply:
yep, done!